packages/base/package.json

```diff
@@ -50,21 +57,57 @@
       "default": "./dist/esm/utils/library/vue/index.js"
     }
   },
-  "scripts": {
-    "build:clean": "rm -rf dist",
-    "build": "tsc --build tsconfig.build.json",
-    "watch": "tsc --watch",
-    "typecheck": "tsc --noEmit",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+  "typesVersions": {
+    "*": {
```

Comment:
What is this sub level?

Reply:
https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#version-selection-with-typesversions

The "*" is acting as a wildcard which means these type definitions apply to all versions of the package. Since we don't need to separate them between versions, we are okay to go with this way.

Following the approach we are doing in `packages/wagmi/package.json`, the following would work as well I think:

```js
"typesVersions": {
  "react": [
    "./dist/types/exports/react/index.d.ts"
  ],
  "vue": [
    "./dist/types/exports/vue.d.ts"
  ]
}
```